src/datasets/arrow_dataset.py

```diff
@@ -341,6 +341,17 @@ def to_tf_dataset(
 
         Returns:
             :class:`tf.data.Dataset`
+
+        Example:
+
+        ```py
+        >>> ds_train = ds["train"].to_tf_dataset(
```

Comment:
well, this might be outdated pretty soon :p
see https://github.com/huggingface/datasets/pull/4170